pfcpiface/parse_sdf.go

```diff
@@ -147,9 +150,13 @@ func parseFlowDesc(flowDesc, ueIP string) (*ipFilterRule, error) {
 				return nil, err
 			}
 
-			if fields[i+1] != "to" {
+			if i+1 < len(fields) && fields[i+1] != "to" {
 				i++
 
+				if i >= len(fields) {
+					return nil, errBadFilterDesc
+				}
```

Comment:
Given that you are checking `i+1 < len(fields)`, is not the following `if` statement redundant?

```suggestion
```